templates/layouts/_base.html

```diff
@@ -36,10 +36,7 @@
 
 {% set pageConfig = {
   "title": full_page_title,
-  "header": page_header,
-  "serviceLinks": service_links,
```

Comment:
Same with `serviceLinks` and `language`, they can stay in initial page config here.